packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -111,7 +118,9 @@ export const convertRemoteParticipantToVideoGalleryRemoteParticipant = (
   state: RemoteParticipantConnectionState,
   displayName?: string,
   /* @conditional-compile-remove(raise-hand) */
-  raisedHand?: unknown // temp unknown type to build stable
+  raisedHand?: unknown, // temp unknown type to build stable
+  /* @conditional-compile-remove(reaction) */
+  reaction?: unknown // temp unknown type to build stable
```

Comment:
note: using unknown should not be necessary here, the conditional compile comment will strip this out of stable builds